frontend/src/components/common/Table.tsx

```diff
@@ -8,51 +8,72 @@ import {
   Tbody,
   Td,
 } from "@chakra-ui/react";
+import Pagination from "./Pagination";
 
 export interface TableRow {
   values: any[];
   menu: React.ReactElement;
 }
 
-interface TableProps {
+export interface TableProps {
```

Comment:
```suggestion
interface TableProps {
```
do you need this change?